lib/css/base/_stacks-configuration-dynamic.less

```diff
@@ -96,16 +72,7 @@
     @button-danger-filled-selected-background-color: var(--red-800);
     @button-danger-filled-number-color:              var(--black-900);
 
-    //  Button Primary
-    @button-primary-color:                           @white;
-    @button-primary-background-color:                var(--theme-secondary-400);
-    @button-primary-hover-color:                     @white;
-    @button-primary-hover-background-color:          var(--theme-secondary-500);
-    @button-primary-active-background-color:         var(--theme-secondary-700);
-    @button-primary-selected-color:                  var(--white);
-    @button-primary-selected-background-color:       var(--theme-secondary-700);
-    @button-primary-number-color:                    var(--theme-secondary-900);
-
+    // TODO not overridden in prod
     //  Tags
     @tags-border:                                    transparent;
```

Comment:
When exploring better theming on tags, I ended up deleting these. #717. I could see this entire file going away.

Reply:
Nice! I'm going to leave these in at the moment so you don't have to solve a merge conflict in your PR. Happy to see these are going away early.